### PR TITLE
docs: Fix wrong CLA link in CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by moonsphere**
CLA in CONTRIBUTING.md is wrong

### CODE REVIEW

**Review by Amber1990Zhang — APPROVED** [review]

**Review by Jamesleons — APPROVED** [review]
LGTM

**Comment by dragonly** [comment]
LGTM

**Review by ichn-hu — APPROVED** [review]
LGTM

**Comment by antJack** [comment]
LGTM

**Review by JmPotato — APPROVED** [review]
LGTM.

**Comment by unbyte** [comment]
**L**ao**G**e**T**ai**M**eng

**Review by crazycs520 — COMMENTED** [review]
LGTM

**Review by tisonkun — APPROVED** [review]
+1 Awesome!

**Review by zz-jason — APPROVED** [review]
LGTM 👍

**Review by hunterlxt — APPROVED** [review]
LGTM

**Review by sundy-li — APPROVED** [review]
LGTM

**Comment by JinLingChristopher** [comment]
LGTM

**Review by hi-rustin — APPROVED** [review]
LGTM!

**Comment by moxih** [comment]
LGTM!

**Comment by MrCroxx** [comment]
LGTM

**Comment by ben1009** [comment]
LGTM

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ OceanBase Database is a community-driven open source project and we welcome all
 
 ## Before you contribute
 
-Before you contribute, please click the **Sign in with Github to agree button** to sign the CLA. You can find an example [here](https://cla-assistant.io/pingcap/tidb?pullRequest=16303).
+Before you contribute, please click the **Sign in with Github to agree button** to sign the CLA. You can find an example [here](https://cla-assistant.io/oceanbase/oceanbase?pullRequest=4).
```
Comment (hi-rustin) [inline CONTRIBUTING.md:7]:
```suggestion
Before you contribute, please click the **Sign in with Github to agree button** to sign the CLA. You can find an example [here](https://cla-assistant.io/oceanbase/oceanbase).
```

**Comment by Yangxiamao** [comment]
LGTM🥰

**Comment by ZHANGWENTAI** [comment]
LGTM!

**Review by hi-rustin — CHANGES_REQUESTED** [review]
We don't need to add parameters.

**Comment by zimulala** [comment]
LGTM

**Comment by lhy1024** [comment]
LGTM

**Comment by gozssky** [comment]
LGTM

**Comment by buggithubs** [comment]
LGTM 💯

File: CONTRIBUTING.md (hunk 1 above)
Comment (tisonkun) [reply CONTRIBUTING.md]: Nice catch!

**Review by liukun4515 — APPROVED** [review]

**Review by Connor1996 — APPROVED** [review]
LGTM

**Comment by Howie59** [comment]
LGTM

**Comment by Handora** [comment]
LGTM

File: CONTRIBUTING.md (hunk 1 above)
Comment (tisonkun) [reply CONTRIBUTING.md]: I think you can submit a new review 😉

**Review by AilinKid — APPROVED** [review]
/LGTM

**Review by XuHuaiyu — APPROVED** [review]
LGTM 👶

**Comment by wshwsh12** [comment]
LGTM

**Review by Youngwb — APPROVED** [review]

**Review by handlerww — APPROVED** [review]
LGTM

**Comment by MartinNose** [comment]
LGTM~

**Review by jyz0309 — APPROVED** [review]

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ OceanBase Database is a community-driven open source project and we welcome all
 
 ## Before you contribute
 
-Before you contribute, please click the **Sign in with Github to agree button** to sign the CLA. You can find an example [here](https://cla-assistant.io/pingcap/tidb?pullRequest=16303).
+Before you contribute, please click the **Sign in with Github to agree button** to sign the CLA. You can find an example [here](https://cla-assistant.io/oceanbase/oceanbase).
```
Comment (dragonly) [inline CONTRIBUTING.md:7]:
```suggestion
Before you contribute, please click the **Sign in with GitHub to agree button** to sign the CLA. You can find an example [here](https://cla-assistant.io/oceanbase/oceanbase).
```
The `h` in `GitHub` should be capitalized.
image

**Comment by suzaku** [comment]
LGTM

File: CONTRIBUTING.md (hunk 2 above)
Comment (hi-rustin) [reply CONTRIBUTING.md:7]: Good catch!

**Review by unbyte — APPROVED** [review]

File: CONTRIBUTING.md (hunk 2 above)
Comment (hi-rustin) [reply CONTRIBUTING.md]: image

It should be capitalized.

**Comment by An-DJ** [comment]
LGTM

**Comment by MizuhaHimuraki** [comment]
Thanks for your feedback. We have fixed it in #9

**Comment by Ehco1996** [comment]
LGTM

**Comment by dragonly** [comment]
> Thanks for your feedback. We have fixed it in #9

My last fix is not included.
And btw you guys just lost 3 contributors :/

**Comment by arronliu1989** [comment]
LGTM

> My last fix is not included.
> And btw you guys just lost 3 contributors :/

now is 4

**Comment by codievilky** [comment]
LGTM

**Comment by jacklightChen** [comment]
LGTM

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -4,7 +4,7 @@ OceanBase Database is a community-driven open source project and we welcome all
 
 ## Before you contribute
 
-Before you contribute, please click the **Sign in with Github to agree button** to sign the CLA. You can find an example [here](https://cla-assistant.io/pingcap/tidb?pullRequest=16303).
```
Comment (4ft35t) [inline CONTRIBUTING.md:7]: LGTM

**Comment by kiyoakii** [comment]
Interesting.

**Comment by 9547** [comment]
lgtm

**Comment by zou-weidong** [comment]
LGTM

**Comment by ClSlaid** [comment]
LGTM

**Comment by tisonkun** [comment]
> Thanks for your feedback. We have fixed it in #9

If I don't get it wrong. 5 < 9?

**Review by jasonliu747 — COMMENTED** [review]
LGTM！

**Comment by konjac** [comment]
LGTM!

**Comment by pentium3** [comment]
LGTM

**Review by pentium3 — COMMENTED** [review]
LGTM

**Comment by weihualiu** [comment]
LGTM

**Review by bobby285271 — COMMENTED** [review]
LGTM

**Comment by gaoran10** [comment]
LGTM

**Comment by reorx** [comment]
LMAO

**Comment by imjoey** [comment]
LGTM

**Comment by chenyukang** [comment]
LGTM

**Comment by hanlins** [comment]
LMAO

**Comment by mc256** [comment]
LGTM

**Comment by zachgenius** [comment]
LGTM, LMFAO

**Comment by Kidsunbo** [comment]
LGTM

**Comment by motecshine** [comment]
LGTM

**Review by Himself65 — COMMENTED** [review]
LGTM

**Comment by fefive-sxh** [comment]
LGTM

**Comment by guobai** [comment]
LGTMo-o

**Comment by luyongjuan** [comment]
LGTM

**Comment by x-ainger** [comment]
LGTM

**Comment by mengxin9014** [comment]
LGTM

**Comment by Xintangtang** [comment]
LGTM

**Review by lilien1010 — COMMENTED** [review]
LGTM

**Comment by mantuliu** [comment]
LGTM

**Comment by shonge** [comment]
LGTM

**Comment by 2014BDuck** [comment]
配合这个知乎提问食用更加..
https://www.zhihu.com/question/362506458/answer/1213152978

**Comment by LINxiansheng** [comment]
> > Thanks for your feedback. We have fixed it in #9
> 
> My last fix is not included.
> And btw you guys just lost 3 contributors :/

Thanks for your feedback, and we will reopen this pr.

**Comment by LINxiansheng** [comment]
Reopen the pr.

**Comment by xuelangos** [comment]
LGTM

**Comment by ICEYSELF** [comment]
LFGTM.

**Comment by jtyjty99999** [comment]
LMAO

**Comment by xlwh** [comment]
LGTM 👍👍👍👍👍👍👍👍👍👍

File: CONTRIBUTING.md (hunk 3 above)
Comment (luolitao) [reply CONTRIBUTING.md]: Let me have a look look.

**Comment by Zheaoli** [comment]
LGTM

**Comment by fafaface** [comment]
LGTM

**Comment by oldthreefeng** [comment]
LGTM

**Comment by tczpl** [comment]
# 打工人怎样依靠考公摆脱社畜命运?
# 一、你要抛弃自己的幻想
你憧憬未来，你会当上高管，可是，打工人的内卷上限更多靠运气还是努力？

你安慰自己，你在创造价值，可是，打工人的剩余价值是不是被股东们分了？

你也有兴趣，你喜欢这工作，可是，打工人的爱好凭啥用绩效和目标来考核？

# 二、你要给自己一个开始
请你不要迷茫，看看身边考公上岸的人，与其羡慕不如就此开始！

你将不再犹豫，不再拖延，先买本考公教材了解，干就完了！

迈出第一步，就从今天开始，给自己的明天一个机会！

# 三、你要勇于以考促练
不必迷信考公机构，多问问备考或上岸的朋友，实践中出真知。

国考省考市考乃至烟草电网，机会真的很多，有合适岗位就都去，就当刷行测申论。

公司太忙没空备考和考试，那可以先换个有空的公司。

考公人，考公魂，上岸都是人上人！

转自 [考公网](KaoGongWang.com) 祝您早日上岸！

**Comment by weimumu** [comment]
@tczpl 考公才能拯救这个世界，才能在不躺平的情况下避免内卷，同时生三个

**Comment by ICEYSELF** [comment]
我就感觉到快，有催人跑的意思

**Comment by tabVersion** [comment]
LGTM.

**Comment by FoxDaxian** [comment]
look good to me

**Comment by lhyaaa** [comment]
LGTM

**Review by laike9m — COMMENTED** [review]
LOL

**Comment by wardenlym** [comment]
LGTM

**Comment by Al-assad** [comment]
LGTM

**Pull request merged**